src/mango/src/mango_cursor.erl

```diff
@@ -583,6 +585,9 @@ ddoc_name(<<"_design/", Name/binary>>) ->
 ddoc_name(Name) ->
     Name.
 
+set_from_list(KVs) ->
```

Comment:
Would not it be better to move this definition to some shared utility module instead of duplicating its definition in multiple modules?  One could include a similar wrapper for `sets:new([{version, 2}])` (as e.g. `set_new/0`) to extend consistency to the creation of sets.

Reply:
Yeah, you're right, it's one line only but it's repeated 4 times, a util function would be nicer here.